src/modules/mail/templates/email.templates.ts

```diff
@@ -13,3 +13,25 @@ export const sendCodeMessage = (userName: string, code: string) => {
           </div>
         `;
 };
+
+export const sendWelcomeMessage = (
+  userName: string,
+  appUrl: string // Recibe la URL base como argumento
+) => `
```

Comment:
can we try to search for more friendly templates?otherwise can u design a better email template? you can take some references in google

Reply:
Yeah of course

Reply:
Sure!

Reply:
![image](https://github.com/user-attachments/assets/aa619ca9-03fe-4207-8d1a-de3749dc365c)

Is it okay like this?